eap-datagrid-subsystem/README.md

```diff
@@ -0,0 +1,100 @@
+Web application: A web application use a cache provided by the EAP server
+==========================================================================================
+Author: Wolf-Dieter Fink
+Level: Beginner
+Technologies: Infinispan, EAP
+
+
+What is it?
+-----------
+
+This example demonstrates how to use the JDG EAP modules and configure an EAP server to manage JDG caches.
+The configuration is included within the EAP standalone.xml file. The application code will get a cache reference
+by injection or lookup via JNDI. The caches are not exposed and only applications deployed on this EAP instance 
+can use it.
+The lifecycle of the cache is managed by the EAP instance, the applications can undeployed or deployed without affects to the cache.
+
+
+
+Prepare the server instances
+-------------
+Simple start
+1.  Create a fresh copy of EAP 7.1 (or newer) server
+
+2.  Unzip jboss-datagrid-7.2.0-eap-modules-library.zip and copy the modules/* directories into the EAP_HOME/modules directory
+
+3.  start EAP with the default configuration standalone.xml
+
+        $EAP_HOME/bin/standalone.sh
+
+4. use the CLI scipts to configure the datagrid subsystem and necessary dependencies
+
+        $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install1-standalone-local.cli
+        > restart the server !
+       $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install2-standalone-local.cli
+
+
+Build and Run the example
+-------------------------
+1. Type this command to build and deploy the archive:
+
+        mvn clean package
+        cp web/target/jboss-eap-datagrid-subsystem-AppWeb.war $EAP_HOME/standalone/deployments
+
+
+Run the web application in your browser
+=======================================
+
+      http://localhost:8080/jboss-eap-datagrid-subsystem-AppWeb/
+
+Add, delete and list entries ...
+
+Run as a clustered cache
+=====================================
+
+The configuration for EAP is a not clustered one, but it is possible to use the install scripts for a *ha* configuration as well.
+If the JDG caches should be clustered the installation scripts in the step above should be install?-standalone-clustered.cli
```

Comment:
`install?-standalone-clustered.cli` should be `install1-standalone-clustered.cli`

Reply:
There are two install scripts for both modes

Reply:
Gotcha, I thought it was a typo. `*` is what I would normally expect for a wildcard.